lib/mail_room/configuration.rb

```diff
@@ -8,17 +8,20 @@ class Configuration
     def initialize(options={})
       self.mailboxes = []
 
-      if options.has_key?(:config_path)
-        config_file = YAML.load_file(options[:config_path])
+      return unless options.has_key?(:config_path)
+      return unless File.exist?(options[:config_path])
+      
+      config_file = YAML.load_file(options[:config_path])
 
-        set_mailboxes(config_file[:mailboxes])
-      end
+      set_mailboxes(config_file[:mailboxes])
     end
 
     # Builds individual mailboxes from YAML configuration
     # 
     # @param mailboxes_config
     def set_mailboxes(mailboxes_config)
+      return unless mailboxes_config
```

Comment:
The reason I added this check and the other one is because in our case, we're shipping GitLab with `config/mail_room.yml.example` which the admin needs to rename to `mail_room.yml` themselves if they want to use reply by email, since they'll need to add their own credentials, which shouldn't be checked into the repository. If the user doesn't want to use reply_by_email at all, `mail_room -c config/mail_room.yml.example` should not fail.

Would it be an option to add a flag to fail silently?